### PR TITLE
debian: remove duplicate agent jar copy

### LINKED ISSUE #4906: Duplicate jar packages are installed under ubuntu
##### ISSUE TYPE

 * Bug Report
##### COMPONENT NAME

~~~
INSTALL
~~~

##### CLOUDSTACK VERSION

~~~
4.15.0.0
~~~

##### OS / ENVIRONMENT

OS： ubuntu 20.04
deb:  cloudstack-agent_4.15.0.0~focal_all.deb

##### SUMMARY

After installing cloudstack-agent_4.15.0.0~focal_all.deb under UBUNTU 20.04, two "same" (same as md5) jar packages are found under /usr/share/cloudstack-agent/lib.

/usr/share/cloudstack-agent/lib
8038649400beafdbf1a32193a9331c8b  cloud-agent-4.15.0.0.jar
8038649400beafdbf1a32193a9331c8b  cloudstack-agent.jar

In CentOS 7, there is only one cloud-agent-4.15.0.0.jar, no cloudstack- agent.jar

##### STEPS TO REPRODUCE

~~~

~~~

##### EXPECTED RESULTS

~~~

~~~

##### ACTUAL RESULTS

~~~

~~~

### PR DESCRIPTION
**Pull request by rhtyd**
The cloud-agent is dependency of the KVM hypervisor plugin, don't
explicitly install/copy it again.

Fixes #4906

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [x] Cleanup (Code refactoring and cleanup, that may add test cases)

### CODE REVIEW

**Comment by rhtyd** [comment]
@blueorangutan package

**Review by shwstppr — APPROVED** [review]
LGTM

File: debian/rules [hunk 1]
```diff
@@ -38,7 +38,6 @@ override_dh_auto_install:
 	mkdir $(DESTDIR)/$(SYSCONFDIR)/profile.d
 	mkdir $(DESTDIR)/usr/share/$(PACKAGE)-agent
 	mkdir $(DESTDIR)/usr/share/$(PACKAGE)-agent/plugins
-	install -D agent/target/cloud-agent-$(VERSION).jar $(DESTDIR)/usr/share/$(PACKAGE)-agent/lib/$(PACKAGE)-agent.jar
 	install -D plugins/hypervisors/kvm/target/cloud-plugin-hypervisor-kvm-$(VERSION).jar $(DESTDIR)/usr/share/$(PACKAGE)-agent/lib/
```
Comment (shwstppr) [inline debian/rules:41]:
```suggestion
	install -D plugins/hypervisors/kvm/target/cloud-plugin-hypervisor-kvm-$(VERSION).jar $(DESTDIR)/usr/share/$(PACKAGE)-agent/lib/cloud-plugin-hypervisor-kvm-$(VERSION).jar
```
@rhtyd build giving error,
```
install -D plugins/hypervisors/kvm/target/cloud-plugin-hypervisor-kvm-4.15.1.0-SNAPSHOT.jar "debian/tmp"/usr/share/cloudstack-agent/lib/
install: target 'debian/tmp/usr/share/cloudstack-agent/lib/' is not a directory: No such file or directory
debian/rules:26: recipe for target 'override_dh_auto_install' failed
```
Adding jar filename should fix this?

**Review by shwstppr — CHANGES_REQUESTED** [review]

File: debian/rules (hunk 1 above)
Comment (rhtyd) [reply debian/rules:42]: @shwstppr thnx, the mkdir was missing fixed.

**Comment by rhtyd** [comment]
@blueorangutan package

**Review by shwstppr — APPROVED** [review]
LGTM. Verified on a Ubuntu 20.04 host,

```
root@pr4927-t474-kvm-ubuntu20-kvm1:~# ll /usr/share/cloudstack-agent/lib/ | grep -i agent
-rwxr-xr-x 1 root root   93390 Apr 16 10:03 cloud-agent-4.15.1.0-SNAPSHOT.jar*
-rwxr-xr-x 1 root root    6678 Apr 16 10:03 cloud-framework-agent-lb-4.15.1.0-SNAPSHOT.jar*
```

**Review by GabrielBrascher — APPROVED** [review]

**Pull request merged**